### PR TITLE
Upgrade to PrestaShop UI KIT v1.2.0

### LINKED ISSUE #17177: [UI Kit] Improvements corrections
Epic: #16587 

[Link to InVision prototype](https://invis.io/S9UXGJA73B5)

List components that haven't been called on the back-office but have been integrated in the UI Kit.

# Alerts

- [x]  Delete "click me to delete"
- [x]  Icon must be aligned with the first text line
- [x]  "Read more" button stays in black
- [x] Sticky warning alert: background #FFFBD3, icon #FAB000
- [x] Verify the button text color: #363A41 ("Read more") 
- [x] Replace all toast alerts by the new design from the UI Kit
- [x] Success toast alert: green #53D572 
- [x] Warning toast alert: orangey yellow #FAB000  
- [ ] Add animation for the new toast alerts

# Cards

- [x]  Fix using cards were it shouldn't be (cf. tables)

→ Need to be ready for 1.8

# Buttons

- [x]  Fix the radius on button in the toolbar
- [x]  Fix icon on dropdown (rotating on click)
- [x]  Rem to be validate
- [x]  No background for outline buttons
- [x]  Button stays active after click on button group (cf. date)

# Radio buttons

- [x]  Fix radius

# Photos

- [x]  Delete radius on pictures (Catalog > Products)

# Placeholders

- [x]  Change italic text by regular (placeholder, search bar)
# Toolbox

- [x]  Padding: 10px
- [x]  Icon size: 19px
- [x]  Show toolbox infos on hover and not on click
- [x] Change the markup to show new toolbox

# Ratios/Spacers

- [ ]  Change by 4px, 8px, 16px, 24px, 32px, 64px

# Toggles

- [x]  Put the right grey when toggle is off: #B3C7CD
- [x]  Replace "YES/NO" buttons by toggles
- [x]  Replace checks and crosses on table by toggles and try to not refresh the page when clicking on toggles (show a toast alert instead)

# Typography

- [x] Update font size: 
image

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | See this epic https://github.com/PrestaShop/PrestaShop/issues/17177
| Type?         | improvement
| Category?     | BO
| BC breaks?    | Does it break backward compatibility? yes/no
| Deprecations? | Does it deprecate an existing feature? yes/no
| Fixed ticket? | #17177
| How to test?  | Check this epic : https://github.com/PrestaShop/PrestaShop/issues/17177, everypoints should looks fine in BO, you need to replace the "prestakit" version in `admin-dev/themes/new-themes/package.json` to `../../../prestashop-ui-kit` this being the path where you got the UIKit installed on your computer, you also need to `npm install` again and `npm run build` after changing the path. ⚠️ This only concerns migrated pages ⚠️ 

I tagged it as a BC break, as some html structure are modified such as `Activate button` on product listing, or dropdowns on module pages, if a module rely on these, it may change his behavior

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17253)

⚠️ ⚠️ We need to update the UIKit before this being merged, but we need QA validated before upgrading the UIKit to avoid too much version on it. Don't merge it until we add a green message at the bottom! ⚠️ ⚠️

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/pages/_modules.scss
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss]:
```suggestion
```

File: src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig]: Why put all the div in the `if` while only the div tag is different

Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig]: Not only the div tag, also the input which should be checked, wdyt ? Better duplicating the condition 3 times or duplicating the markup ? I think you're right, the condition should be duplicated, this makes the markup more maintainable

Comment (Progi1984) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig]: Exact

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
What I found so far (sorry, noted this in french, so it's mainly for me) :

- [x] Probleme avec les icônes input error/warning + background-colour au hover
- [x] Select2 dropdown => soucis de selecteur
- [x] Dropdown module => miss use of markup
- [x] Positions, Link widget, Localization (language, currencies), Taxes, Param g contact, traffic see, advanced (Team, Webservice)  => rotate au dropdown a retirer
- [x] Param general button
- [x] Alignements icônes listes
- [x] Uninstall module alert title is buggy

**Comment by sarahdib** [comment]
In modules -> catalog module :
Capture d’écran 2020-06-18 à 11 21 30

In apparence -> position 
Capture d’écran 2020-06-18 à 11 29 14

In International -> translation : 
Capture d’écran 2020-06-18 à 11 56 32

In Payement : (mbo)
Capture d’écran 2020-06-18 à 11 49 04

Missing round on all dropdown : 
Capture d’écran 2020-06-18 à 11 13 41

Tooltip on click not on hover :
https://drive.google.com/file/d/13hR2fLHXlWDBRkKikY99rgz2t5jC4-zu/view

The button isn't visible : 
https://drive.google.com/file/d/1LH6iEwidVwJk3KCugcXz65jQytkAt5BV/view

Some checkbox and dropdown don't use bootstrap
Capture d’écran 2020-06-18 à 11 11 27

Capture d’écran 2020-06-18 à 12 34 14

**Comment by NeOMakinG** [comment]
The button isn't visible :
https://drive.google.com/file/d/1LH6iEwidVwJk3KCugcXz65jQytkAt5BV/view

It's a backend issue, the button is not checked to "No" if no is selected !

**Comment by NeOMakinG** [comment]
What about markups that are not using bootstrap ? Like checkbox and other select ? Should we switch these to bootstrap ? That's maybe harder that we think if this is not thought at the beginning @PrestaShop/prestashop-core-developers

**Comment by NeOMakinG** [comment]
And finally, MBO things should be fixed on MBO, did my best to do it on the core but we should keep separate some styles that are already inside MBO !

**Comment by Progi1984** [comment]
@NeOMakinG Ready for review ?

**Comment by NeOMakinG** [comment]
I asked 2 question that needs to be responded

**Comment by eternoendless** [comment]
> What about markups that are not using bootstrap ? Like checkbox and other select ? Should we switch these to bootstrap ?

I think we should homogenize all controls, but it can be done later.

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig]:
```suggestion
            {{ 'Enabled modules'|trans({}, 'Admin.Modules.Feature') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig]:
```suggestion
            {{ 'Disabled modules'|trans({}, 'Admin.Modules.Feature') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig]:
```suggestion
        {{ 'All categories'|trans({}, 'Admin.Modules.Feature') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig]:
```suggestion
        data-category-display-name="{{ 'Recently used'|trans({}, 'Admin.Modules.Feature') }}"
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig]:
```suggestion
        {{ 'Recently used'|trans({}, 'Admin.Modules.Feature') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig:55]: I can see "trans(" and a domain (Admin.Modules.Feature) but I cannot see any wording. Is it normal?

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig]:
```suggestion
          {{ 'All categories'|trans({}, 'Admin.Modules.Feature') }}
```

**Comment by NeOMakinG** [comment]
@Julievrz you sure about all these feedbacks ? I didn't add or modify any wording, these are existing since years and not related to this PR at all

wdyt @PrestaShop/prestashop-maintainers ?

**Comment by Julievrz** [comment]
> @Julievrz you sure about all these feedbacks ? I didn't add or modify any wording, these are existing since years and not related to this PR at all
> 
> wdyt @PrestaShop/prestashop-maintainers ?

I talked about it with @LouiseBonnard on Slack but for now she is still double-checking my wording reviews so I need to wait for her approval before applying the Wording label. :)

**Comment by LouiseBonnard** [comment]
I agree with @Julievrz, it makes no sense to keep useless and irrelevant capital letters. Even if the wordings already exist. ;-)

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig:55]: The wording is coming from the backend, wdyt @PrestaShop/prestashop-maintainers ?

**Review by atomiix — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (atomiix) [reply src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig:55]: That’s because the string being translated is dynamically passed to the translator. So you can only say if the domain is right

**Review by Progi1984 — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig
Comment (Julievrz) [reply src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig:55]: Mhm, I'm not sure to understand how I can say if the domain is right without the wording? 🤔 I can read "subMenu.name" so if it is a "title" then yes it should be in Admin.Modules.Feature

**Comment by sarahdib** [comment]
- Alert : 
the icon and the text are not aligned
Capture d’écran 2020-11-06 à 14 32 19

- Order
Capture d’écran 2020-11-06 à 15 26 12

- Order detail
Round is missing 
Capture d’écran 2020-11-06 à 15 10 19
Payment block 
Capture d’écran 2020-11-06 à 15 18 16

- Invoices
Capture d’écran 2020-11-06 à 15 35 13

- Product
Should be blue
Capture d’écran 2020-11-06 à 16 03 10
Capture d’écran 2020-11-06 à 16 13 33

Should be align
Capture d’écran 2020-11-06 à 16 06 22

- Monitoring 
Should be on-off switch
Capture d’écran 2020-11-06 à 16 38 55

Brand & Supplier
Should be on-off switch
Capture d’écran 2020-11-06 à 16 49 38

- Stock & Mouvement 
Should be rounded
Capture d’écran 2020-11-06 à 17 14 02
Capture d’écran 2020-11-06 à 17 17 04
Capture d’écran 2020-11-06 à 17 17 44

- Customer settings 
Should be on-off switch
Capture d’écran 2020-11-06 à 17 19 23

The radio button is not the ui kit one 
Capture d’écran 2020-11-06 à 17 22 16

Dropdown should be closer
Capture d’écran 2020-11-06 à 17 23 59

The icon is a too big
Capture d’écran 2020-11-06 à 17 30 07


- Theme & Logo 
Capture d’écran 2020-11-06 à 17 39 13

- Email theme
The icon should be after the text
Capture d’écran 2020-11-06 à 17 46 45

- Pages
Should be on-off switch
Capture d’écran 2020-11-06 à 17 48 54

- Currencies 
https://drive.google.com/file/d/1k_8vpXKEvTcesttVTfkUVwTBK6WxoGsd/view

- Taxes
Should be on-off switch
Capture d’écran 2020-11-06 à 18 00 14

- Translation 
https://drive.google.com/file/d/1XediCVv0I-A7MF_gsmtzHQcMyx1NIBzw/view

- Email
image (5)


- Team
https://drive.google.com/file/d/1dfVcfbrGdrLCbPa6pFa1GBmduLugpdK5/view

**Comment by sarahdib** [comment]
Bugs are not critical, I approved this PR and I will create issues for all this bugs

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by PululuK** [comment]
Good job @NeOMakinG 
Thank you

**Pull request merged**
